examples/jupyter/mdgxs-part-i.ipynb

```diff
@@ -1141,6 +1137,31 @@
     "plt.legend(legend, loc=1, bbox_to_anchor=(1.55, 0.95))"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[CellFilter\n",
+       " \tBins           =\t[1]\n",
+       " \tID             =\t3,\n",
+       " DelayedGroupFilter\n",
+       " \tBins           =\t[1 2 3 4 5 6]\n",
+       " \tID             =\t13]"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dr_tally.filters"
+   ]
+  },
```

Comment:
Was this addition intentional?

Reply:
Nope. Not sure how that snuck in there. Thanks!